build.gradle

```diff
@@ -294,3 +294,41 @@ release {
     pushToRemote = ''
   }
 }
+
+// Reports linkage errors across multiple Apache Beam artifact ids.
+//
+// To use (from the root of project):
+//    ./gradlew -Ppublishing -PjavaLinkageArtifactIds=artifactId1,artifactId2,... :checkJavaLinkage
+//
+// For example:
+//    ./gradlew -Ppublishing -PjavaLinkageArtifactIds=beam-sdks-java-core,beam-sdks-java-io-jdbc :checkJavaLinkage
+//
+// Note that this task publishes artifacts into your local Maven repository.
+if (project.hasProperty('javaLinkageArtifactIds')) {
```

Comment:
Why do we need to set this property?

Reply:
I don't think we want to perform linkage checking across all artifacts at the same time since there is no expectation that runner A is compatible with runner B. I was envisioning that there would be a common set of linkage checks that could be run such as `{runner A, runner B, runner C} x {core libs + IOs}`

Reply:
But wouldn't these be `runnerA:checkLinkage`, `runnerB:checkLinkage` etc?

Reply:
Ah, I misread how you were using this property. But it would seem nice to base it on the current project's runtime scope.